Convert CTA links to blue

stephaniehobson opened the pull request:
https://github.com/mozilla/protocol/issues/280

alexgibson commented on src/assets/sass/protocol/components/_language-switcher.scss:
```diff
@@ -12,6 +12,8 @@
     .mzp-c-cta-link {
         @include bidi(((margin, 0 $margin-lg $margin-sm 0, 0 0 $margin-sm $margin-lg),));
         @include text-body-sm;
+        border-bottom: none;
```
This selector doesn't appear to be in master anymore since the new footer landed, does this branch need a rebase?

alexgibson commented on src/assets/sass/protocol/base/elements/_links.scss:
```diff
@@ -30,17 +30,17 @@
-        border-bottom: 2px solid $color-black;
-        color: $color-black;
+        border-bottom: 2px solid;
+        color: $color-link-hover;
```
Not a blocker: I'm guessing that the hover states for the light and dark themes differ is intentional, but would making them more consistent be worth doing? e.g. the color doesn't change, but we do remove the bottom-border?

stephaniehobson commented:
Updated with excellent standardization suggestions. Rebased and double checked language selector did not have any problems in the rebase.

Added links to t-mzp-dark examples so we won't forget the dark themes when updating links in the future (hopefully).

Pushed to demo1
- https://demo1--mozilla-protocol.netlify.com/patterns/atoms/links.html
- https://demo1--mozilla-protocol.netlify.com/demos/call-out.html
- https://demo1--mozilla-protocol.netlify.com/demos/card-layout.html
- https://demo1--mozilla-protocol.netlify.com/demos/feature-card.html

Also verified they remove the need for custom link styling on firefox/accounts and firefox/fffy.

stephaniehobson commented:
Added to changelog.
